### PR TITLE
Update problem descriptions in 'Measurements'

### PR DESCRIPTION
**Pull request by wsgac**
Update descriptions of problems 1.8 and 1.9 to reflect the fact that
bits1 and bits2 can differ in the first dimension.

### CODE REVIEW

File: Measurements/Tasks.qs [hunk 1]
```diff
@@ -134,8 +134,9 @@ namespace Quantum.Kata.Measurements {
     //      1) N qubits which are guaranteed to be
     //         in one of the two superposition states described by the given arrays of bit strings.
     //      2) two arrays of bit strings represented as Bool[][]s.
-    //         Each of the arrays has a size M ⨯ N and consists of M bit strings of length N
-    //         (the first index is the index of the bit string, the second - the index of the qubit).
+    //         The arrays have dimensions M₁ ⨯ N and M₂ ⨯ N respectively, where N is the number of
+    // 	       qubits and M₁ and M₂ can be different (the first index is the index of
+    // 	       the bit string, the second - the index of the qubit).
```
Comment (tcNickolas) [inline Measurements/Tasks.qs]: Could you please use 4 spaces instead of a tab for indentation?

File: Measurements/Tasks.qs [hunk 2]
```diff
@@ -134,8 +134,9 @@ namespace Quantum.Kata.Measurements {
     //      1) N qubits which are guaranteed to be
     //         in one of the two superposition states described by the given arrays of bit strings.
     //      2) two arrays of bit strings represented as Bool[][]s.
-    //         Each of the arrays has a size M ⨯ N and consists of M bit strings of length N
-    //         (the first index is the index of the bit string, the second - the index of the qubit).
+    //         The arrays have dimensions M₁ ⨯ N and M₂ ⨯ N respectively, where N is the number of
+    // 	       qubits and M₁ and M₂ can be different (the first index is the index of
```
Comment (tcNickolas) [inline Measurements/Tasks.qs]: I'd say "and M₁ and M₂ are the numbers of bit strings in each array. Note that in general M₁ ≠ M₂" - this way we explain what they are, not just that they are different.

File: Measurements/Tasks.qs (hunk 1 above)
Comment (wsgac) [reply Measurements/Tasks.qs]: Done. My Emacs did not have the proper settings for Q#.

File: Measurements/Tasks.qs (hunk 2 above)
Comment (wsgac) [reply Measurements/Tasks.qs]: Done

**Pull request merged**